test/com/google/enterprise/adaptor/database/ResponseGeneratorTest.java

```diff
@@ -63,18 +72,25 @@ public void testMissingColumnNameForFilepathMode() {
   @Test
   public void testMissingColumnNameForBlobMode() {
     thrown.expect(NullPointerException.class);
-    ResponseGenerator.blobColumn(Collections.<String, String>emptyMap());
+    ResponseGenerator.contentColumn(Collections.<String, String>emptyMap());
```

Comment:
Something should call blobColumn (preferably without thrown an exception, though). Maybe add a single test that asserts that blobColumn(...).getClass().getName() equals "com.google.enterprise.adaptor.database.ContentColumn"?